README.md

```diff
@@ -32,10 +32,11 @@ Outside of the SDK repository simply run `cargo new my-robot` this will create a
 
 ### Configure Cargo.toml
 The details of Cargo.toml are beyond the scope of this README please refer to [this](https://doc.rust-lang.org/cargo/reference/manifest.html) for more info
-Add the following to your Cargo.toml :
+Add the following to your Cargo.toml, we'll be using the `viam-rust-utils` package to connect to the robot:
 ``` toml
 [dependencies]
 viam = {path = "../"} #this path should point to the Rust SDK repository
+viam-rust-utils = { git = "https://github.com/viamrobotics/rust-utils.git", rev = "e177376c084bdadf8d67d077629fb16b3cd72097" }
```

Comment:
I think `rust-utils` is updated at this point in crates.io, so this should probably be `viam-rust-utils = "0.0.18"`